api/openapi.yaml

```diff
@@ -481,8 +537,8 @@ components:
       type: object
       required:
         - subject
-        - expiresAt
-        - token
+        # - expiresAt
```

Comment:
WHy is this commented out?

Reply:
The OpenAPI generator doesn't support readOnly, writeOnly: https://github.com/drwpow/openapi-typescript/issues/604